tsl/test/isolation/expected/cagg_insert.out

```diff
@@ -1,141 +1,199 @@
-Parsed test spec with 12 sessions
+Parsed test spec with 17 sessions
 
-starting permutation: LockInvalThrEx Refresh2 Refresh UnlockInvalThrEx
+starting permutation: invalidation_threshold_insert_wait invalidation_threshold_cagg_1 invalidation_threshold_cagg_2 invalidation_threshold_insert_release
+step invalidation_threshold_insert_wait: SELECT debug_waitpoint_enable('invalidation_threshold_insert');
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step invalidation_threshold_cagg_1: 
+  CREATE MATERIALIZED VIEW cagg_1
+  WITH (timescaledb.continuous) AS
+  SELECT time_bucket('5', time), COUNT(location)
+  FROM ts_continuous_test_1
+  GROUP BY 1
+  WITH NO DATA;
+
+step invalidation_threshold_cagg_2: 
+  CREATE MATERIALIZED VIEW cagg_2
+  WITH (timescaledb.continuous) AS
+  SELECT time_bucket('5', time), COUNT(location)
+  FROM ts_continuous_test_1
+  GROUP BY 1
+  WITH NO DATA;
+ <waiting ...>
```

Comment:
It is not because during the trigger creation we check if the trigger already exists. Have a look here: https://github.com/timescale/timescaledb/blob/main/tsl/src/continuous_aggs/create.c#L334

Reply:
wait a moment... I found the issue... and this is something not related to this PR... will check it more carefully next week and maybe provide a fix for it... did some small tests and using the `OR REPLACE` option for CreateTrigStatement make it work like a charm but this is something available since pg14.

Reply:
related PR to fix create trigger race: #6001